spec.md

```diff
@@ -410,7 +412,7 @@ how Platforms might expose these values to their users.
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| id | string | The id of the Oauth client that the dashboard will use. If present, MUST be a non-empty string. |
+| id | string | The id of the OAuth client that the dashboard will use. If present, MUST be a non-empty string. |
```

Comment:
I would take changes that are unrelated to this PR and make them into different PRs, or they'll be waiting in limbo for this to merge.